src/app/admin/students/services/students.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { GetStudentsInterface } from '../interfaces/get-students-interface';
+import { CreareUpdateStudentsInterface } from '../interfaces/creare-update-students-interface';
+import { ResponseInterface } from '../interfaces/response-interface';
+
+@Injectable()
+export class StudentsService {
+
+  private createStudentUrl = 'Student/insertData';
+  private readStudentUrl = 'Student/getStudentsByGroup';
+  private updateStudentUrl = 'Student/update';
+  private deleteStudentUrl = 'Student/del';
+
+  constructor(private httpClient: HttpClient) {}
+
+  getStudentsByGroup(id: number): Observable<GetStudentsInterface[] & ResponseInterface> {
+    return this.httpClient.get<GetStudentsInterface[] & ResponseInterface>(`${this.readStudentUrl}/${id}`);
```

Comment:
Will we have GetStudentsInterface array and string response property as response here?

Reply:
Yes, the idea was to have array as a response but in case in case if something goes wrong we case response property as it would be probably more convenient to work with it if the request failed for some reasons